mne/gui/tests/test_stc.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-clause
+
+import os.path as op
+import numpy as np
+from numpy.testing import assert_allclose
+
+import pytest
+
+import mne
+from mne.datasets import testing
+from mne.io.constants import FIFF
+
+data_path = testing.data_path(download=False)
+subject = 'sample'
+subjects_dir = op.join(data_path, 'subjects')
+fname_raw = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc_raw.fif')
+fname_fwd_vol = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc-meg-vol-7-fwd.fif')
+fname_fwd = op.join(
+    data_path, 'MEG', 'sample', 'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
```

Comment:
New code should probably prefer `/` using Path syntax but I can live with `op.join` if needed